docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM debian:buster AS stage_build
 
-ARG EMSCRIPTEN_VERSION=1.39.18
+ARG EMSCRIPTEN_VERSION=1.39.20
```

Comment:
Perhaps we could change this default value to `tot` (tip-of-tree build) to catch these issues earlier?

Another idea would be to pass on this value here:
https://github.com/emscripten-core/emsdk/blob/56f5a2bcc55576a29f17e9d14ed25ce82a341b6c/.circleci/config.yml#L100
```diff
-          command: docker build --network host ./docker
+          command: docker build --network host  --build-arg EMSCRIPTEN_VERSION=tot ./docker
```
(untested)

Reply:
The only disadvantage of the above approach is that it is necessary to execute the `./emsdk update tags` command, which increases the Docker image size by 6.8 MB. This won't be a problem if we only do this in CI, I guess.

Reply:
Hmm .. that is sad .. we should probably fix `update tags` to just grab the latest tags from master rather than what I imagine it is doing which a full git clone of emscripten-releases.

Reply:
I've opened https://github.com/emscripten-core/emsdk/pull/571 for discussion. I noticed there's already a `--shallow` parameter for the `install` command, perhaps that should also be supported for the `update-tags` command?

Reply:
You can also just run `rm -rf releases` after `update-tags` for now if you like.